gulpfile.mjs

```diff
@@ -2250,7 +2250,7 @@ function packageJson() {
     bugs: DIST_BUGS_URL,
     license: DIST_LICENSE,
     optionalDependencies: {
-      "@napi-rs/canvas": "^0.1.62",
+      "@napi-rs/canvas": "^0.1.63",
```

Comment:
Please upgrade to `0.1.64`, as `0.1.63` contains a regression that may cause an abnormal exit when using text operations on Windows: https://github.com/Brooooooklyn/canvas/pull/953, `0.1.62` does not have this issue, and it was accidentally introduced in `0.1.63` when adding support for the `riscv` platform.

Reply:
Thanks for the heads-up; done in the new commit.